velox/common/memory/SharedArbitrator.cpp

```diff
@@ -34,8 +38,22 @@ uint64_t maxGrowBytes(const MemoryPool& pool) {
 uint64_t capacityAfterGrowth(const MemoryPool& pool, uint64_t targetBytes) {
   return pool.capacity() + targetBytes;
 }
+
+bool isFactoryInitialized = []() {
```

Comment:
Why this is used?

Reply:
It should be `isFactoryRegistered`. It's for registering the shared implementation by default.